bin/emqx

```diff
@@ -349,10 +349,17 @@ case "$1" in
         if ! relx_nodetool "stop"; then
             exit 1
         fi
-        while $(kill -s 0 "$PID" 2>/dev/null);
-        do
-            sleep 1
+        WAIT_TIME=${WAIT_FOR_ERLANG:-15}
```

Comment:
why the wait? some comments to explain ?

Reply:
Because occasionally there will be `while $(kill -s 0 "$PID" 2>/dev/null)` infinite loop, but I found that adding `WAIT_TIME` will cause problems with github actions, so I am going to undo this modification